data/json/recipes/other/materials.json

```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```

Comment:
I do believe there is a perl packing tool (packs an interpreter, module and script) as an exe. It might be worth distributing such a binary with the project?